src/sentry/static/sentry/app/plugins/components/pluginIcon.tsx

```diff
@@ -1,5 +1,6 @@
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
+import aws from 'app/../images/logos/logo-aws.svg';
```

Comment:
Grouped wrong, you'll get eslint complaints

Reply:
good call